sodetlib/noise.py

```diff
@@ -540,7 +543,9 @@ def take_noise(S, cfg, acq_time=30, plot_band_summary=True, nbins=40,
     if save_dir is None:
         save_dir = S.plot_dir
 
-    sid = sdl.take_g3_data(S, acq_time)
+    if g3_tag is None:
+        g3_tag = "oper,noise"
```

Comment:
Do you imagine the g3_tag var to be set by users to describe what they're doing? Like adding a tag "transition_with_hwp" to describe a noise timestream?

If so maybe we should change this so it appends to the tag list instead of overwriting it. If we decide to do this the tags should probably be stored in as a set until it's time to write them out to avoid duplicating the 'oper' tag.